src/api/services/MessagesService.js

```diff
@@ -2,14 +2,46 @@ import axios from '../http';
 
 const MessagesService = {
   fetchMessages: async ({ pageNumber }) => {
-    const response = await axios.get(
-      `/messages?offset=${pageNumber - 1}&pagesize=15`
-    );
-    return response.data;
+    try {
+      const response = await axios.get(
+        `/messages?offset=${pageNumber - 1}&pagesize=15`
+      );
```

Comment:
Extract this to a variable
```suggestion
      const response = await axios.get(url);
```